src/FsAutoComplete.Core/Commands.fs

```diff
@@ -44,6 +44,15 @@ type Commands (serialize : Serializer) =
                 ()
         } |> Async.Start
 
+    let updateSymbolUsesCache (filename: FilePath) (check: FSharpCheckFileResults) =
+        async {
+            try
+                let! symbols = check.GetAllUsesOfAllSymbolsInFile()
+                SymbolCache.sendSymbols serialize filename symbols
+            with
+            | _ -> ()
+        }
```

Comment:
This whole thing does not look very diagnostic friendly :)

Reply:
TBF, I don't think I've ever seen `GetAllUsesOfAllSymbolsInFile` to throw exception, but yes there are couple of places in FSAC where we just silently ignore exceptions and that should be changed.